src/main/java/com/sinfolix/Sai_Samarth/controller/ProductCatlogueController.java

```diff
@@ -42,6 +42,18 @@ public ResponseEntity<ApiResponse> deleteProductCatlogue(@PathVariable int id){
         this.productCatlogueService.deleteProductCatlogue(id);
         return ResponseEntity.ok(new ApiResponse("User Deleted Successfully",true));
     }
+
+//    DISABLE disable the product
+    @PatchMapping("/{id}/disable")
+    public void disableProduct(@PathVariable int id){
+        productCatlogueService.disableProduct(id);
+    }
+
+//    ADD COMMENT add comment
+    @PatchMapping("/{id}/comment")
+    public ProductCatlogue addComment(@PathVariable Integer id, @RequestBody String comment){
+        return productCatlogueService.addComment(id,comment );
```

Comment:
We should not add comment separately. It should be taken while giving order by customer. Also image section is missing in changes